BlueShift-iOS-SDK/InApps/BlueShiftInAppNotificationHelper.m

```diff
@@ -92,7 +92,8 @@ + (CGFloat)getPresentationAreaHeightForWindow:(UIWindow*)window {
         topMargin = [[UIApplication sharedApplication] statusBarFrame].size.height;
         bottomMargin = [[UIApplication sharedApplication] statusBarFrame].size.height;
     }
-    CGFloat presentationAreaHeight = window.bounds.size.height - topMargin - bottomMargin;
+    CGFloat windowHeight = [self getApplicationWindowSize:window].height;
```

Comment:
the window size will always be present right? do we need any nill checks? I saw the method def, just double checking.

Reply:
Yes, a valid size will be returned all the time.